.github/workflows/eudonet_paris_import.yml

```
@@ -0,0 +1,89 @@
+name: Eudonet Paris Import
+
+# on:
+#   schedule:
+#     - cron: '0 17 * * 1' # Tous les lundis à 17h00
+
+on:
+  push:
+    branches:
+      - feat/eudonet-auto
+
+jobs:
+  eudonet_paris_import:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Setup PHP with PECL extension
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.2'
+
+      - name: Get Composer Cache Directory
+        id: composer-cache
+        run: |
+          echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
+
+      - uses: actions/cache@v3
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-composer-
+
+      - name: Install Scalingo CLI
+        run: curl -O https://cli-dl.scalingo.com/install && bash install
+
+      - name: Install SSH key
+        # Credit: https://stackoverflow.com/a/69234389
+        run: |
+          install -m 600 -D /dev/null ~/.ssh/id_rsa
+          echo "${{ secrets.GH_SCALINGO_SSH_PRIVATE_KEY }}" > ~/.ssh/id_rsa
+
+      - uses: actions/cache@v3
+        id: addok-bundle-cache
+        with:
+          path: docker/addok/addok-data
+          key: ${{ runner.os }}-addok-bundle-${{ secrets.EUDONET_PARIS_KDRIVE_FILE_ID }}
+
+      - name: Download and unzip Addok bundle
+        if: steps.addok-bundle-cache.outputs.cache-hit != 'true'
+        run: |
+          mkdir -p tmp
+          bash tools/download_addok_bundle.sh tmp/addok-archive.zip
+          unzip -d tmp/addok-archive tmp/addok-archive.zip
+          unzip -d docker/addok/addok-data tmp/addok-archive/addok-dialog-bundle.zip
+        env:
+          EUDONET_PARIS_KDRIVE_TOKEN: ${{ secrets.EUDONET_PARIS_KDRIVE_TOKEN }}
+          EUDONET_PARIS_KDRIVE_FILE_ID: ${{ secrets.EUDONET_PARIS_KDRIVE_FILE_ID }}
+
+      - name: Start Addok
+        run: |
+          make addok_start
+          ./tools/wait_for_url.py --interval 5 --max-attempts 10 "http://localhost:7878/search?q=rue+de+la+concertation"
+
+      - name: Init environment variables
+        run: |
+          echo "DATABASE_URL=${{ secrets.EUDONET_PARIS_IMPORT_DATABASE_URL_PR }}" >> .env.local
```

TODO avant de merger

```suggestion
          echo "DATABASE_URL=${{ secrets.EUDONET_PARIS_IMPORT_DATABASE_URL }}" >> .env.local
```